README.md

```diff
@@ -237,6 +262,13 @@ Connecting a server will automatically pause all requests so that you can attach
 
 **Note: Only requests that are launched after a server is connected/made debug server can be attached.**
 
+Here are a couple curl commands that can be used with the test-app to start sample requests:
```

Comment:
I don't think we should include any examples in the README that depend on the test-app. The test-app is for mlxprs developers and should only be referenced in the CONTRIBUTING. If we need examples that depend on an application being deployed to MarkLogic, we should have an e.g. "./examples/(some name)" project that the README provides instructions on deploying and trying out. (This is also out of scope of the JIRA ticket and thus better addressed via a separate ticket/PR).

Reply:
Removed